src/MeetingsSDKAdapter/controls/JoinControl.js

```diff
@@ -0,0 +1,37 @@
+import {Observable, of} from 'rxjs';
+import {MeetingControlState} from '@webex/component-adapter-interfaces';
+import MeetingControl from './MeetingControl';
+
+/**
+ * Display options of a meeting control.
+ *
+ * @external MeetingControlDisplay
+ * @see {@link https://github.com/webex/component-adapter-interfaces/blob/master/src/MeetingsAdapter.js#L58}
+ */
+
+export default class JoinControl extends MeetingControl {
+  /**
+   * Attempts to join the meeting of the given meeting ID.
+   * If the meeting is successfully joined, a ready event is dispatched.
+   *
+   * @param {string} meetingID  Id of the meeting to join
+   */
+  async action(meetingID) {
+    await this.adapter.joinMeeting(meetingID);
+  }
+
+  /**
+   * Returns and observable that emits the display of the control.
+   *
+   * @returns {Observable.<MeetingControlDisplay>} Observable stream that emits display of the join control
```

Comment:
I would leave this empty as it's obvious from the type and the description of the function itself, but if you want to keep the description, remove the "stream" word: Observable that emits display data